kolibri/plugins/learn/templates/learn/learn.html

```diff
@@ -10,6 +10,19 @@
     model.synced = true;
     var collection = {{kolibri}}.resources.ContentNodeResource.getCollection({parent: root_node_pk}, JSON.parse("{{ nodes|escapejs }}"));
     collection.synced = true;
+
+    // Get list of channels.
+    var channels = JSON.parse("{{ channels | escapejs }}");
+    var defaultChannelId = "7199dde6-95db-4ee4-ab39-2222d5af1e51"; // TODO: API also fetches user-selected default channel.
```

Comment:
The hard coded channel_id in `views.py` was just to make it work with the dummy databases I created. Similar to what @jamalex [did in views.py](https://github.com/learningequality/kolibri/pull/328/files#diff-bae452d1e37f9e753fda753d8c25990aL16). Here I hard coded `defaultChannelId` because inconsistency with the channel_id not having dashes. But in the future channel_id will have dashes, correct @jamalex?